Skuid_Techniques/Multi_Part_Form/README.md

```diff
@@ -0,0 +1,33 @@
+# Multi-part Form Navigation and Real-time Completion Status
+
+Skuid can be used to create very effective complex data-entry forms.  There are two principles that can help users work through an extensive data-entry process:  
+- Break up the form into chunks.
+- Provide wayfinding and progress status information. 
+
+The example below uses the Skuid wizard component to break up a form into several different sections, but also uses the vertical navigation component - with selected-state conditions - to provide more effective wayfinding and status information for the whole form.  The example shows how to integrate the vertical navigation items and the wizard buttons so that both components stay in synch with user action. 
+
+The example also shows how to include dynamic information within a navigation item.  This could be a counter of completed questions, or a completion icon, etc.    
+
+<img src="MultiPartForm.png" width="300"></img>
+
+## Instructions
+- Page API:  V2
+- Data source: Only UI-only models are used. 
+- Design system: [Download this Design System file](SamplePages.designsystem).  Use the Import function on the Design System page to add this system to your org.  
+- Page XML:  [Copy the XML from this page](Multi_Part_Form.xml), or save it as an XML file, and upload it as a new page in your Salesforce Org.  
+
+## Notes
+
+Synching the wizard and the vertical navigation selected state:
+
+- A UI Only Field is used to manage selected state in the vertical navigation component.   (See the `NavPrimary` field in the `UI Control` Model).   All navigation items and wizard buttons have action steps that update this field to an appropriate value. 
+- The Navigation Items and the Wizard buttons each have an wizard component action that moves the user to the correct step. 
+
+Including dynamic information in navigation step: 
+
+- The vertical navigation component will allow html in item labels.  This allows for formatting flexibility,  as well as the inclusion of merge syntax - to push model data into the label. 
+- Because the component is not connected to a particular model - this has to be global merge syntax  - like `{{$Model.FormDetail.data.0.status1}}`.
```

Comment:
```suggestion
- Because the component is not connected to a particular model this has to be global merge syntax, like `{{$Model.FormDetail.data.0.status1}}`.
```

The global merge note is less of an aside than the dashes would suggest. Let's use commas instead.